core/cap-TBD.md

```diff
@@ -0,0 +1,562 @@
+## Preamble
+
+```
+CAP: To Be Assigned
```

Comment:
We should consider having this CAP and all its sub-caps share the same number to make the relations obvious. E.g. calling this CAP-0046 and the other ones CAP-0046-a, CAP-0046-b, CAP-0046-c etc. 
I know this doesn't match the current indexing rules but we are dealing with a new CAP structuring process.

Reply:
(I don't know if we are supposed to touch the CAP index after it's being proposed before being finalized.)

Reply:
Oh, sure, I guess. I talked to @MonsieurNicolas about this today and he liked the sound of it, so I guess I will do that in this PR as well.